content/blog/2019/07/2019-07-24-azure-artifact-manager.adoc

```diff
@@ -0,0 +1,109 @@
+* *Azure Container Name:* The container under which to keep your artifacts. If the container name does not exist in the blob, this plugin automatically creates one for you when artifacts are uploaded to the blob.
+* *Base Prefix:* Prefix added to your artifact paths stored in your container, a forward slash will be parsed as a folder. In the following screenshot, all your artifacts will be stored in the “staging” folder in the container “Jenkins”.
+
+image:/images/post-images/2019-07-24-azure-artifact-manager/2.configuration.png[role=center]
+
+====== New Jenkins instance
+
+If you need to create a new Jenkins master, follow this https://docs.microsoft.com/en-us/azure/jenkins/install-jenkins-solution-template[tutorial] to quickly create an Jenkins instance on Azure. In the *Integration Settings* section, you can now set up Azure Artifact Manager directly. Note that you can change any of the configuration after your Jenkins instance is created. Azure storage account and credential, in this case, are still prerequisites.
+
+image:/images/post-images/2019-07-24-azure-artifact-manager/3.integration-setting-azure.png[role=center]
+
+== Usage
+
+====== Jenkins Pipeline
+
+Here are a few commonly used artifact related steps in pipeline jobs; all are supported to push artifacts to the Azure Storage blob specified.
+
+You can use _archiveArtifacts_ step to archive target artifacts into Azure storage. For more details about _archiveArtifacts_ step, see the Jenkins https://jenkins.io/doc/pipeline/steps/core/#archiveartifacts-archive-the-artifacts[archiveArtifacts setp documentation].
+
+----
+node {
+  //...
+  stage('Archive') { 
+    archiveArtifacts "pattern"
+  } 
+}
+----
+
+You can use the _unarchive_ step to retrieve the artifacts from Azure storage. For more details about _unarchive_ step, please see https://jenkins.io/doc/pipeline/steps/workflow-basic-steps/#code-unarchive-code-copy-archived-artifacts-into-the-workspace[unarchive step documentation].
+
+----
+node {
+  //...
+  stage('Unarchive') { 
+    unarchive mapping: ["pattern": '.']
+  } 
+}
+----
+
+To save a set of files so that you can use them later in the same build (generally on another node or workspace), you can use _stash_ step to store files into Azure storage for later use. _Stash_ step documentation can be found https://jenkins.io/doc/pipeline/steps/workflow-basic-steps/#unstash-restore-files-previously-stashed[here].
+
+
+----
+node {
+  //...
+  stash name: 'name', includes: '*'
+}
+----
+
+You can use _unstash_ step to retrieve the files saved with _stash_ step from Azure storage to the local workspace.  _Unstash_ documentation can be found https://jenkins.io/doc/pipeline/steps/workflow-basic-steps/#unstash-restore-files-previously-stashed[here].
+
+----
+node {
+  //...
+  unstash 'name'
+}
+----
+
+
+====== FreeStyle Job
```

Comment:
```suggestion
=== FreeStyle Job
```

Reply:
🎉Thank you, @zbynek! So glad checks are all ✅